### PR TITLE
Add Notes in Create Pipeline Using Jenkinsfile/Editing Panels-v3.1

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by Felixnoo** [comment]
@kubesphere/sig-devops Please help review the updates I made in this PR when you are available, thanks.

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md
Comment (LinuxSuRen) [inline content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md:104]: Please hold on about this part. We're still under discussion about this. See also https://github.com/kubesphere/kubesphere/issues/3639

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md
Comment (LinuxSuRen) [inline content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md:251]: There're two cases here:

* First, anyone who can run this pipeline, then he can approve or reject it if there are no specific approvers
* Second, same as you described above

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md]: I'll remove this part in the PR.

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md]: Thanks you the comment and I made some changes.

**Comment by Sherlock113** [comment]
@LinuxSuRen Hi, do you have any comments about this pr?

**Comment by Felixnoo** [comment]
@LinuxSuRen Hi, please take a look at this PR when you are available.

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**
